desc/vmec.py

```diff
@@ -100,9 +103,29 @@ def load(cls, path, L=-1, M=-1, N=-1, spectral_indexing="fringe"):
         inputs["pressure"] = np.zeros((preset, 2))
         inputs["pressure"][:, 0] = np.arange(0, 2 * preset, 2)
         inputs["pressure"][:, 1] = file.variables["am"][:] * p0
-        inputs["iota"] = np.zeros((preset, 2))
-        inputs["iota"][:, 0] = np.arange(0, 2 * preset, 2)
-        inputs["iota"][:, 1] = file.variables["ai"][:]
+        if profile == "iota":
+            inputs["iota"] = np.zeros((preset, 2))
+            inputs["iota"][:, 0] = np.arange(0, 2 * preset, 2)
+            inputs["iota"][:, 1] = file.variables["ai"][:]
+            inputs["current"] = None
+        if profile == "current":
```

Comment:
Is it worth printing here that DESC is assuming the default current mode in VMEC (I'(s))? as technically in VMEC you can specify the I(s) profile if you pass a certain flag

Reply:
do we know what that flag is? If so we could detect it and throw an error or convert it?

Reply:
I think it's `pcurr_type`. If `pcurr_type = power_series`, then it's expecting the derivative profile. If `pcurr_type = power_series_I`, then it's expecting the I(s) profile.

Reply:
Actually in the input reader right now we just throw a warning, do we want to do the same, or just skip the integration if its ``pcurr_type = power_series_I``?

Reply:
Checking the input type would be "better", but I think just throwing a warning is fine for now.